client/src/components/body/landingpage/sitemap.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import "../../../assets/stylesheets/components/App.scss"
+import "../../../assets/stylesheets/pages/LandingPage.scss"
+import {connect} from "react-redux";
+import {bindActionCreators} from 'redux';
+import {actions as mainActions} from "../../../reducers/MainReducer";
+
+const mapDispatchToProps = (dispatch) => ({
+    actions: bindActionCreators(mainActions, dispatch)
+});
+
+const handleLab = (actions, lab) => {
```

Comment:
You can remove this and the redux stuff like dispatch to props, since we are navigating via URL